Formula/texlive.rb

```diff
@@ -0,0 +1,499 @@
+  end
+
+  resource "Net::SSLeay" do
+    url "https://cpan.metacpan.org/authors/id/C/CH/CHRISN/Net-SSLeay-1.90.tar.gz"
+    sha256 "f8696cfaca98234679efeedc288a9398fcf77176f1f515dbc589ada7c650dc93"
+  end
+
+  resource "IO::Socket::SSL" do
+    url "https://cpan.metacpan.org/authors/id/S/SU/SULLR/IO-Socket-SSL-2.072.tar.gz"
+    sha256 "b5bee81db3905a9069340a450a48e1e1b32dec4ede0064f5703bafb9a707b89d"
+  end
+
+  resource "LWP::Protocol::https" do
+    url "https://cpan.metacpan.org/authors/id/O/OA/OALDERS/LWP-Protocol-https-6.10.tar.gz"
+    sha256 "cecfc31fe2d4fc854cac47fce13d3a502e8fdfe60c5bc1c09535743185f2a86c"
+  end
+
+  resource "Tk" do
+    url "https://cpan.metacpan.org/authors/id/S/SR/SREZIC/Tk-804.036.tar.gz"
+    sha256 "32aa7271a6bdfedc3330119b3825daddd0aa4b5c936f84ad74eabb932a200a5e"
+  end
+
+  resource "Pygments" do
+    url "https://files.pythonhosted.org/packages/b7/b3/5cba26637fe43500d4568d0ee7b7362de1fb29c0e158d50b4b69e9a40422/Pygments-2.10.0.tar.gz"
+    sha256 "f398865f7eb6874156579fdf36bc840a03cab64d1cde9e93d68f46a425ec52c6"
+  end
+
+  def install
+    # Install Perl resources
+    ENV.prepend_create_path "PERL5LIB", libexec/"lib/perl5"
+    ENV["PERL_MM_USE_DEFAULT"] = "1"
+    ENV["OPENSSL_PREFIX"] = Formula["openssl@1.1"].opt_prefix
+
+    tex_resources = ["texlive-extra", "install-tl", "texlive-texmf"]
+    python_resources = ["Pygments"]
+
+    resources.each do |r|
+      r.stage do
+        next if tex_resources.include? r.name
+
+        next if python_resources.include? r.name
+
+        if File.exist? "Makefile.PL"
+          system "perl", "Makefile.PL", "INSTALL_BASE=#{libexec}",
+            "CCFLAGS=-I#{Formula["freetype"].opt_include}/freetype2"
+          system "make"
+          system "make", "install"
+        else
+          system "perl", "Build.PL", "--install_base", libexec
+          system "./Build"
+          system "./Build", "install"
+        end
+      end
+    end
+
+    # Install Python resources
+    venv = virtualenv_create(libexec, "python3")
+    venv.pip_install resource("Pygments")
+
+    # Install TeXLive resources
+    resource("texlive-extra").stage do
+      share.install "tlpkg"
+    end
+
+    resource("install-tl").stage do
+      cd "tlpkg" do
+        (share/"tlpkg").install "installer"
+        (share/"tlpkg").install "tltcl"
+      end
+    end
+
+    resource("texlive-texmf").stage do
+      share.install "texmf-dist"
+    end
+
+    # Clean unused files
+    rm_rf share/"texmf-dist/doc"
+    rm_rf share/"tlpkg/installer/wget"
+    rm_rf share/"tlpkg/installer/xz"
+
+    # Set up config files to use the correct path for the TeXLive root
+    inreplace buildpath/"texk/kpathsea/texmf.cnf",
+      "TEXMFROOT = $SELFAUTOPARENT", "TEXMFROOT = $SELFAUTODIR/share"
+    inreplace share/"texmf-dist/web2c/texmfcnf.lua",
+      "selfautoparent:texmf", "selfautodir:share/texmf"
+
+    # Fix path resolution in some scripts (these should be upstreamed)
+    pathfix_files = [
```

Comment:
Generally I think we use 
```suggestion
    pathfix_files = %w[
```

Reply:
Can't add comments in-line if you use `%w[`.

Reply:
I'll leave the array with inline comments as is, but the file lists can be changed to `%w`, because I don't think we'll need comments.  I'll push the changes and let me know if they need fixing.